.eslintrc

```diff
@@ -135,6 +136,15 @@
     "no-undef": 2,
     "no-var": 2,
     "no-warning-comments": [2, { "terms": ["do not submit"], "location": "anywhere" }],
+    "notice/notice": [2,
+      {
+        "mustMatch": "Copyright 20\\d{2} The AMP HTML Authors\\.",
+        "templateFile": "build-system/artifacts/eslint-notice-license-template.txt",
+        "messages": {
+          "whenFailedToMatch": "Mising or incorrect license header. Run `gulp lint --fix` to add automatically."
```

Comment:
Three changes to this line:

1. Drop the ``Run `gulp lint --fix` to add automatically.`` part. We already print a message for that at the end of `gulp lint`.

https://github.com/ampproject/amphtml/blob/b84350059273cec5d8e683b3bfae073cf356dcd1/build-system/tasks/lint.js#L120-L129

2. For consistency with other error messages, drop the period at the end of `Mising or incorrect license header.`
3. `s/Mising/Missing/` :)

Reply:
Missing `s` ;)